.travis.yml

```diff
@@ -12,7 +12,6 @@ branches:
     - master
 
 env:
-  - TARGET="56"
```

Comment:
I assume this has to be dropped because tests were not passing with elasticsearch-php for 5.6?

The reason I'm asking is because I would like to keep the tests for 5.6 around as long as possible, even if we don't official support it. But if it becomes a hassle, lets remove it. I probably have to check if es-php is really not compatible with 5.6 or if it just removed it from the composer file.